tests/unix_listener.rs

```diff
@@ -187,6 +190,57 @@ where
     handle.join().unwrap();
 }
 
+#[test]
+fn unix_listener_multiple_accepts() {
```

Comment:
Current implementation on Windows fails this test. Subsequent `accept` calls after the first to the same listener hang because `accept` is not considered an I/O operation and won't send event when ready.

I've fixed this test by wrapping `accept` in `do_io` in `net/uds/listener.rs`